src/Microsoft.AspNetCore.Http.Connections/Internal/HttpConnectionStatus.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+
```

Comment:
nit: remove newline

Reply:
remove and sort seems to always leave these sitting around :(